Gruntfile.js

```diff
@@ -201,6 +201,8 @@ module.exports = function (grunt) {
         src: ['<%= yeoman.app %>/index.html'],
         ignorePath:  /\.\.\//,
         exclude: [
+          // bower registration error? we get 2x versions of uri.js/urijs
```

Comment:
Will do, I believe in my original PR's I had this bower PR based on the npm PR, I may have copied over extra stuff.  Once it merges, I'll rebase this PR & ensure its correct.